cmd/benchmark-utils_test.go

```diff
@@ -316,7 +317,7 @@ func runPutObjectBenchmarkParallel(b *testing.B, obj ObjectLayer, objSize int) {
 	// obtains random bucket name.
 	bucket := getRandomBucketName()
 	// create bucket.
-	err = obj.MakeBucketWithLocation(bucket, "")
+	err = obj.MakeBucketWithLocation(context.Background(), bucket, "")
```

Comment:
init context at start of function and pass it to this and subsequent objectlayer api calls.

Reply:
context.Background() in tests is just a dummy, will not matter if we pass same context of different,=.